src/main/java/org/broadinstitute/hellbender/engine/dataflow/transforms/composite/AddContextDataToReadOptimizedFn.java

```diff
@@ -0,0 +1,280 @@
+package org.broadinstitute.hellbender.engine.dataflow.transforms.composite;
```

Comment:
Whoops, it looks like you've accidentally resurrected an old dataflow package. Can you move this to the `engine.spark` package, and rename to `AddContextDataToReadSparkOptimized`?